backend.py

```diff
@@ -80,10 +83,44 @@ def rotate(self, where_to):
         """
         Rotate robot according to a given direction.
         """
-
         self.direction = self.direction.get_new_direction(where_to)
 
 
+    def apply_card_effect(self, state):
+        """
+        Get the current card (depending on game round) and perform the card effect.
+        If the card's effect is move - it calls robot's method walk,
+        if it is rotation - robot's method rotate.
+
+        TODO: resolve card's priority
+        """
+        # card on an index of a current game round
+        current_card = self.program[state.game_round - 1]
+
+        if isinstance(current_card, MoveCard):
+            self.walk(current_card.distance, state)
+
+        if isinstance(current_card, RotateCard):
+            self.rotate(current_card.rotation)
+
+
+class Card:
+    def __init__(self, priority, value):
```

Comment:
Co je `value` u obecné karty?

Reply:
Nic. Tento atribut se nastavuje u objektu podtřídy. Můžu ho zde vynechat?

Reply:
Asi jo, zkus to! :)

Reply:
It works!